scripts/mapd-deps-ubuntu1604.sh

```diff
@@ -11,6 +11,12 @@ source $SCRIPTS_DIR/common-functions.sh
 sudo mkdir -p $PREFIX
 sudo chown -R $(id -u) $PREFIX
 
+apt install wget -y
+
+# add llvm 6 repository
+add-apt-repository -y 'deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial main'
```

Comment:
Https would be better, though it might require installing another package first (apt-transport-https). Slight chance add-apt-repository also needs to be installed.

Further down, do you need to specify exactly which version of clang and llvm to install? I thought this repo included all recent versions, so would have to explicitly say to install *-6.0.